torch/jit/__init__.py

```diff
@@ -65,7 +65,7 @@ def _parse_env(name, default, true_message, false_message):
 # destruction order issues.
 _python_cu = torch._C.CompilationUnit()
 
-Future = torch._C.Future
+Future = Future
```

Comment:
Add a comment that this is here to satisfy `flake8` so someone doesn't delete it for looking weird